pkg/local_object_storage/metabase/metadata.go

```diff
@@ -191,9 +202,25 @@ type SearchCursor struct {
 	ValIDOff int
 }
 
+// splits VAL_DELIM_OID.
+func splitValOID(b []byte) ([]byte, []byte, error) {
+	if len(b) < oid.Size {
```

Comment:
`< oid.Size + attributeDelimiterLen`? And `+1` to avoid "missing value" check.

Reply:
simplified